src/test/java/co/rsk/federate/adapter/ThinConverterTest.java

```diff
@@ -34,12 +32,12 @@ void toThinInstanceStoredBlock() {
         int height = 200;
         org.bitcoinj.core.Block originalBlock = new org.bitcoinj.core.Block(params, 1, createOriginalHash(), createOriginalHash(), 1000, 2000, 3000, new ArrayList<>());
         org.bitcoinj.core.StoredBlock originalStoredBlock = new org.bitcoinj.core.StoredBlock(originalBlock, chainWork, height);
-        co.rsk.bitcoinj.core.StoredBlock thinStoredBlock = ThinConverter.toThinInstance(originalStoredBlock, BridgeRegTestConstants.getInstance());
+        co.rsk.bitcoinj.core.StoredBlock thinStoredBlock = ThinConverter.toThinInstance(originalStoredBlock, new BridgeRegTestConstants());
```

Comment:
same here

Reply:
I agree. It doesn't collide with any other class with the name name in this file, so we can simply put `StoredBlock`.

Reply:
We have 
```
org.bitcoinj.core.StoredBlock originalStoredBlock
co.rsk.bitcoinj.core.StoredBlock thinStoredBlock
```
It's important to distinguish between the 2. ThinConverter class is used to convert from bitcoinj-thin to bitcoinj-core and viceversa